client/mass/charts.js

```diff
@@ -174,7 +173,7 @@ function getChartTypeList(self, state) {
 			clickTo: self.loadChartSpecificMenu
 		},
 		{
-			label: 'Hierarchical Clustering',
+			label: 'Gene Expression',
 			chartType: 'hierCluster',
```

Comment:
see my comment in hierCluster.config.js, not reasonable to still associate `hierCluster` as chart type for this button. should follow logic of metabolite